src/TwigComponent/tests/Fixtures/templates/components/Alert.html.twig

```diff
@@ -0,0 +1,22 @@
+{% props color = 'blue', size = 'md', class = '' %}
+
+{{ cva('alert', {
+    'base': 'rounded-lg',
+    'variants': {
+        'color': {
+            'blue': 'text-blue-800 bg-blue-50 dark:bg-gray-800 dark:text-blue-400',
+            'red': 'text-red-800 bg-red-50 dark:bg-gray-800 dark:text-red-400',
+            'green': 'text-green-800 bg-green-50 dark:bg-gray-800 dark:text-green-400',
+            'yellow': 'text-yellow-800 bg-yellow-50 dark:bg-gray-800 dark:text-yellow-400',
+        },
+        'size': {
+            'sm': 'px-4 py-3 text-sm',
+            'md': 'px-6 py-4 text-base',
+            'lg': 'px-8 py-5 text-lg',
+        }
+    }
+}) }}
```

Comment:
```suggestion
{% cva('alert', {
    'base': 'rounded-lg',
    'variants': {
        'color': {
            'blue': 'text-blue-800 bg-blue-50 dark:bg-gray-800 dark:text-blue-400',
            'red': 'text-red-800 bg-red-50 dark:bg-gray-800 dark:text-red-400',
            'green': 'text-green-800 bg-green-50 dark:bg-gray-800 dark:text-green-400',
            'yellow': 'text-yellow-800 bg-yellow-50 dark:bg-gray-800 dark:text-yellow-400',
        },
        'size': {
            'sm': 'px-4 py-3 text-sm',
            'md': 'px-6 py-4 text-base',
            'lg': 'px-8 py-5 text-lg',
        }
    }
}) %}
```

Reply:
Much harder to do but yes maybe I gonna be have to do it that way

Reply:
Why is it much harder to do? (it's not a trap, that's a real question :d)

Reply:
If you need any help @WebMamba poke me, but i'd also prefer we keep Twig standard concerning tags/functions, ... and if your CSV does not echo anything i would be better to use a tag.